### PR TITLE
fix static checks

### PR DESCRIPTION
**Pull request by alphadose**

### CODE REVIEW

**Review by Aceix — CHANGES_REQUESTED** [review]

File: nmproxy/packet/packet_router.go [hunk 1]
```diff
@@ -133,22 +132,22 @@ func StartRouter() error {
...
-		fmt.Println("Protocol: ", ip.Protocol)
-		fmt.Println()
-
-	}
-	// Check for errors
-	if err := packet.ErrorLayer(); err != nil {
-		fmt.Println("Error decoding some part of the packet:", err)
-	}
-}
+// func printPktInfo(packet gopacket.Packet, inbound bool) {
```
[the remark this answers is not in the document]
Comment (alphadose) [reply nmproxy/packet/packet_router.go:135]: useful for debugging should the need arise

File: functions/localport.go [hunk 2]
```diff
@@ -85,17 +84,17 @@ func UpdateLocalListenPort() error {
...
-	var err error
-	if ncutils.IsMac() { // if node is Mac (Darwin) get the tunnel name first
-		deviceiface, err = local.GetMacIface(address.IP.String())
-		if err != nil || deviceiface == "" {
-			deviceiface = ifacename
-		}
-	}
-	return deviceiface
-}
+// func getRealIface(ifacename string, address net.IPNet) string {
```
[the remark this answers is not in the document]
Comment (abhishek9686) [reply functions/localport.go:87]: this func can be removed if unused

[the remark this answers is not in the document]
Comment (alphadose) [reply functions/localport.go]: done

**Review by Aceix — APPROVED** [review]

**Review by theguy951357 — APPROVED** [review]
ran static check on repo.
no errors
pass

**Pull request merged**